### ISSUE TITLE
[QUEST] Changing the Wind's Course

### ISSUE THREAD

**Issue by ke5hxf** [WoWManiaUK/Redemption#1132]
Changing the Wind's Course
[http://www.wowhead.com/quest=13058/changing-the-winds-course](url)

after completing the chain to get to this part of the quest, 

if you die after losing the effect of The Lorehammer  [http://www.wowhead.com/item=42918/the-lorehammer](url) ,

you are unable to resurrect at the spot of your death. You must return to the Spirit and take a resurrection sickness debuff.

I have tried this several times with the same result.

Client: WoW Mania
Launcher: used and current

basically, trying to fight North Wind, you lose health on your 'mount' Stormhoof, once that's gone, you return to your normal self, and die pretty quickly. The spirit is close by and when you return, the Resurrect button appears, and clicks, but nothing happens. You fly back to the spirit and must take the rez sickness to return to life. Seems your character is phased from your spirit because of the LoreHammer effect. and not being updated properly.
